utils/getOffchainPrice.ts

```diff
@@ -32,6 +32,10 @@ function _getBitstampPriceFromJSON(jsonData: any) {
   return Number(jsonData.last);
 }
 
+function _getCoingeckoPriceFromJSON(jsonData: any) {
+  return Number(jsonData.dextf.usd);
```

Comment:
Could this be generalized instead of being only for dextf?

Reply:
Yeah absolutely. No other contract uses CG here though, so didn't feel necessary to figure out. Thoughts? Think it needs to be generalized?

Reply:
I think it could be named `_getDextfPriceFromJSON` for now and then generalized to `_getCoingeckoPriceFromJSON` once we have multiple contracts that need CoinGecko prices?

Reply:
Either that, or add a `todo` comment that this should be generalized once more contracts are using CoinGecko.

Reply:
Added a todo comment. Thanks for bringing this up. I'll revisit later.